src/plugins/enzymes/src/FindEnzymesDialog.cpp

```diff
@@ -753,21 +763,20 @@ static const QString TOOLTIP_SPACE = "<sub>&nbsp;</sub>";
 
 EnzymeTreeItem::EnzymeTreeItem(const SEnzymeData& ed)
     : enzyme(ed) {
-    auto tt = generateEnzymeTooltip();
-    setText(0, enzyme->id);
-    setData(0, Qt::ToolTipRole, tt);
-    setCheckState(0, Qt::Unchecked);
-    setText(1, enzyme->accession);
-    setData(1, Qt::ToolTipRole, tt);
-    setText(2, enzyme->type);
-    setData(2, Qt::ToolTipRole, tt);
-    setText(3, enzyme->seq);
-    setData(3, Qt::ToolTipRole, tt);
-    setText(4, enzyme->organizm);
-    setData(4, Qt::ToolTipRole, enzyme->organizm);
+    setText(Column::Id, enzyme->id);
+    setData(Column::Id, Qt::ToolTipRole, enzyme->id);
```

Comment:
Is there any reason to duplicate item text in the tooltip? The tooltip is useless in this case. May be show a unified tooltip for the whole row with complete details?

Reply:
I just thought that it's convinient